extensions/amp-fit-text/amp-fit-text.md

```diff
@@ -31,6 +30,13 @@ limitations under the License.
 
 Expands or shrinks its font size to fit the content within the space given to it.
 
+[tip type="note"]
```

Comment:
This should be auto generated.

Reply:
Awesome this is auto generated! Removed the section.